gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -439,6 +441,45 @@ gum_stalker_finalize (GObject * object)
   G_OBJECT_CLASS (gum_stalker_parent_class)->finalize (object);
 }
 
+static GumVirtualizationRequirements
+gum_exec_block_virtualize_linux_sysenter_insn (GumExecBlock * block,
+                                               GumGeneratorContext * gc)
```

Comment:
Function should be moved right after the function using it. (Newspaper ordering.)

Reply:
Fixed